modules/create-wif-cluster-ocm.adoc

```diff
@@ -55,14 +55,23 @@ Workload Identity Federation (WIF) is only supported on {product-title} version
 .. Select a cloud provider region from the *Region* drop-down menu.
 .. Select a *Single zone* or *Multi-zone* configuration.
 +
-.. Optional: Select *Enable Secure Boot support for Shielded VMs* to use Shielded VMs when installing your cluster. For more information, see link:https://cloud.google.com/security/products/shielded-vm[Shielded VMs].
+.. Optional: Select *Enable Secure Boot for Shielded VMs* to use Shielded VMs when installing your cluster. Once you create your cluster, the *Enable Secure Boot for Shielded VMs* setting cannot be changed. For more information, see link:https://cloud.google.com/security/products/shielded-vm[Shielded VMs].
 +
 [IMPORTANT]
 ====
 To successfully create a cluster, you must select *Enable Secure Boot support for Shielded VMs* if your organization has the policy constraint `constraints/compute.requireShieldedVm` enabled. For more information regarding GCP organizational policy constraints, see link:https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints[Organization policy constraints].
 ====
+// +
+// [IMPORTANT]
+// ====
+// Once a machine pool is saved, the *Enable Secure Boot support for Shielded VMs* setting cannot be changed.
```

Comment:
is this a commented out with just notes for you? If not I would replace "saved" with "created"